src/main/resources/db/migration/V5__add_created_and_updated_columns.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE draft_document
```

Comment:
We could do these two things (created, updated) in one go, but only if migration time matters here